custom_components/localtuya/__init__.py

```diff
@@ -71,7 +71,7 @@
 
 from .common import TuyaDevice
 from .config_flow import config_schema
-from .const import DATA_DISCOVERY, DOMAIN, TUYA_DEVICE
+from .const import CONF_PASSIVE_DEVICE, DATA_DISCOVERY, DOMAIN, TUYA_DEVICE
```

Comment:
Shouldn't we add the "passive_device" option in the YAML sample in __init.py__ ? 
I'd also document it in the README.md and info.md.
Great work!

Reply:
Right, I missed that part when I split up the previous PR. Will fix that. Documentation is certainly needed. Maybe we should start looking at GitHub pages for documentation, it's pretty ok (I use it for pyatv over at [pyatv.dev](https://pyatv.dev)).